jobs/kubelet/templates/config/gce.gcfg.erb

```diff
@@ -0,0 +1,5 @@
+<% if p("cloud-provider.type") == "gce" %> 
```

Comment:
Also, did you consider creating this file as part of the `kubeconfig` job and sharing it across these jobs? (Or perhaps creating new shared job)